File: integration-test/src/test/java/com/promise/integrationtest/server/message/ServerGroupMessage.java
package com.promise.integrationtest.server.message;

public enum ServerGroupMessage
{
    DeleteDefault ("Server.Message.ServerGroupDeleteDefault");
    private String id;

    ServerGroupMessage(String id)
    {
        this.id = id;
    }

    public String getId()
    {
        return id;
    }
}

File: server/object/message/ServerGroupMessage.go
package message

import (
	"promise/common/category"
	"promise/common/object/constValue"
	commonMessage "promise/common/object/message"
)

const (
	// MessageServerGroupDeleteDefault Message ID
	MessageServerGroupDeleteDefault = "Server.Message.ServerGroupDeleteDefault"
)

// NewDeleteDefaultServerGroup will return a message.
func NewDeleteDefaultServerGroup() commonMessage.Message {
	ret := commonMessage.NewMessage(category.ServerGroup)
	ret.ID = MessageServerGroupDeleteDefault
	ret.Severity = constValue.SeverityWarning
	ret.Description = "Delete default server group is not allowed."
	return ret
}
